packages/accounts-2fa/2fa-server.js

```diff
@@ -0,0 +1,123 @@
+import { Accounts } from 'meteor/accounts-base';
+import twofactor from 'node-2fa';
+import QRCode from 'qrcode-svg';
+import { Meteor } from 'meteor/meteor';
+
+Accounts.is2faEnabledForUser = selector => {
+  if (!Meteor.isServer) {
+    throw new Meteor.Error(
+      400,
+      'The function is2faEnabledForUser can only be called on the server'
+    );
+  }
+
+  if (typeof selector === 'string') {
+    if (!selector.includes('@')) {
+      selector = { username: selector };
+    } else {
+      selector = { email: selector };
+    }
+  }
+
+  const user = Meteor.users.findOne(selector) || {};
+  const { twoFactorAuthentication } = user;
+  return (
+    twoFactorAuthentication &&
+    twoFactorAuthentication.secret &&
+    twoFactorAuthentication.type === 'otp'
+  );
+};
+
+Accounts.generate2faToken = secret => twofactor.generateToken(secret);
```

Comment:
Why is this method exposed in the Accounts object?

Reply:
This one we can change if there is a better alternative, but I did this way so I could valid on flows in the tests https://github.com/meteor/meteor/blob/64701fe9465da2a3906a9179be175eb7c7b7ecc8/packages/accounts-base/accounts_tests_setup.js#L1-L15

Reply:
If it is only available for tests we should name it starting with `_` so we can change it later without notifying everybody or considering a breaking change.

Reply:
Done